### DISCUSSION TITLE
How to use the API Request/Response code block?

### DISCUSSION THREAD

**Discussion by kengjungwa** [outline/outline#1633] [Self Hosting] [answered]
Hi,

Thanks for the great product. Our dev team really like it. 

From outline doc at https://www.getoutline.com/developers#tag/Revisions/paths/~1revisions.info/post, We saw many API block i.e. Response sample with JSON and HTTP code tab as shown in the following snapshot.
![image](https://user-images.githubusercontent.com/1743535/98528730-4a884d80-22af-11eb-994d-0b0d8c8ad68e.png)

The question is how can we use this block in the document too. Does it require 3rd-party integration? 
Currently, we tried using / to open context menu but could find anything close to it.
![image](https://user-images.githubusercontent.com/1743535/98528855-760b3800-22af-11eb-90c3-5bb7b81c92ac.png)

Any advice is much appreciated. Thanks!!

**Comment by tommoor** [comment, marked as answer]
Hey, I'm sorry I don't know what you mean at all. The editor and the API are completely separate things, perhaps you could describe what you're trying to achieve?

**Reply by kengjungwa** [reply]
Hi @tommoor, thank you for the quick response and sorry for the confusion. Please let me try again.

We are not trying to connect to outline via an API. Basically, my team wants to use outline as a documentation tool for our internal APIs. We like what we see in the outline's document at this page https://www.getoutline.com/developers#tag/Revisions/paths/~1revisions.info/post. 

In the page, there are black, interactive blocks of UI representing information regarding API i.e. url, json and http response code tabs (200, 400, 401, 403) as shown in the first image. 

Now we tried to use the editor to create the similar UI block. As shown in the second image, our best guest was to use the / command to bring up the menu to insert a specific-purpose UI block. However, we couldn't find the one we were looking for.

So my question is how to create a page content with the UI block mentioned above.

Thanks a lot in advance!

**Reply by tommoor** [reply]
I see, the developer documentation is part of a static website, it does not use Outline itself. The website is built on Next.js, the documentation styling comes from a dependency called "redoc", the site is actually open source:
https://github.com/outline/website/blob/develop/pages/developers.tsx

The closest thing in Outline would be a `code` block

**Reply by kengjungwa** [reply]
Thanks a lot tommoor. The information is very helpful :)
